added method to handle the set and get bandwidth mode using Gemini AI

mbonaglia opened the pull request:
as soon as the branch is merged can you delete it or send me an email and i'll do it?

mbonaglia committed:
added method to handle the set and get bandwidth mode using Gemini AI

alfiopuglisi commented:
The implementation in the KURIOSVB1 device is OK, but a new abstract method in the base class is problematic, because it must then be added to all devices, and also in the client, and most devices are not interested in the bandwidth mode.
For device-specific parameters, it's probably better a generic setParameter/getParameter() method like in pysilico_server. I can implement it and we can reduce this PR to just the implementation in the tunable filter device

g-carla committed:
Added get_bandwidth_mode and set_bandwidth_mode methods:
Implemented these methods in plico_motor_server/devices/KURIOSVB1_thorlabs.py to send the appropriate serial commands (BW?, BW=) to the Kurios device.
Added corresponding methods to plico_motor_server/controller/controller.py to handle the client RPC requests for these functions.
Fixed Method Signatures:
Updated get_bandwidth_mode and set_bandwidth_mode in both KURIOSVB1_thorlabs.py and controller.py to accept the axis argument passed by the client RPC call, resolving initial TypeError exceptions. The axis is now correctly passed down from the controller to the device layer.
Corrected Serial Commands:
Fixed the termination characters in the READ_BW and WRITE_BW constants in KURIOSVB1_thorlabs.py (changed \\r to \r), resolving serial communication timeouts.
Fixed Logging Call:
Changed self._logger.info to self._logger.notice in KURIOSVB1_thorlabs.py's set_bandwidth_mode method to fix an AttributeError related to the logger object.

mbonaglia commented:
i updated again the package since the last one was lacking a few items. please refer to this specific version since i tested it and it is working with the hardware in the lab...
